### PR TITLE
feat(web/applications): search results frontend

### PR DESCRIPTION
**Pull request by lutamKC**
## Describe your changes

-add view for the search results page, with table and pagination
- update 'govuk-frontend' to latest version. this leads to some changes in the way the css are imported and some snapshots were updated due to a new attribute aria-live="polite" added in the html

## Issue ticket number and link
https://pins-ds.atlassian.net/browse/BOAS-197

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/applications/search/__tests/applications-search.test.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+const successFullResponse = {
+	page: 1,
+	pageSize: 1,
+	pageCount: 1,
+	itemCount: 1,
+	items: [
+		{
+			id: 76,
+			reference: null,
+			modifiedDate: '1662373959',
```
Comment (mayya-bond-kc) [inline apps/web/src/server/applications/search/__tests/applications-search.test.js:18]: would this be a string or int?

File: apps/web/src/server/views/applications/components/status-tag.component.njk [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	{% case 'Examination' %}
+		{% set statusTagColor = 'purple' %}
+	{% case 'Recommendation' %}
+		{% set statusTagColor = 'yellow' %}
+	{% case 'Withdrawn' %}
+		{% set statusTagColor = 'orange' %}
+	{% endswitch %}
+
+	<strong class="govuk-tag govuk-tag--{{statusTagColor}}">{{ statusLabel }}</strong>
+{% endmacro %}
```
Comment (mayya-bond-kc) [inline apps/web/src/server/views/applications/components/status-tag.component.njk:25]: Just checking, are we reusing these colours in the search results as well as the dashboards?

**Review by mayya-bond-kc — APPROVED** [review]
LGTM

**Review by francescobrenci-AR — APPROVED** [review]

File: apps/web/src/server/views/applications/components/status-tag.component.njk (hunk 2 above)
Comment (lutamKC) [reply apps/web/src/server/views/applications/components/status-tag.component.njk:25]: yes we are

File: apps/web/src/server/applications/search/__tests/applications-search.test.js (hunk 1 above)
Comment (lutamKC) [reply apps/web/src/server/applications/search/__tests/applications-search.test.js:18]: an int usually. not important for this test though

**Pull request merged**